polling_stations/assets/custom_css/style.scss

```diff
@@ -45,20 +45,3 @@ select[size], select[multiple] {
   background-color: $dc-green;
   color: #FFF;
 }
-
-
```

Comment:
I don't actually see these classed being used in the file where you've deleted the donation link, which makes me wonder whether they are in use somewhere else  (and should be kept) if at all (and safe to delete)?

Reply:
`grep -r 'donate_card'` in the root of the repository doesn't show anything, so should be safe to go. I think line 1 of  polling_stations/templates/fragments/donate.html was the only place they were used.